tests-e2e/ocw-ci-test-www/featured_courses.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { test, expect } from "@playwright/test"
+import { WwwPage } from "../util"
+
+test("Course Card lists Instructors", async ({ page }) => {
+  const www = new WwwPage(page)
+  await www.goto()
+  const card = www.getCourseCard("OCW CI Test Course")
+  await expect(card).toContainText(
+    "Instructors(s): Prof. Tester One, Dr. Tester Two, Another Tester Three"
```

Comment:
@Wassaf-Shahzad Here's the e2e test I was thinking of adding from your work in #969!

Previously there were no featured courses in ocw-ci-test-www`. 😦